WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewView.swift

```diff
@@ -0,0 +1,252 @@
+                        .fontWeight(.bold)
+                        .titleStyle()
+
+                    // Subtitle label.
+                    Text(Localization.subtitle)
+                        .foregroundColor(.secondary)
+                        .bodyStyle()
+                }
+                .padding(.bottom, Layout.titleBlockBottomSpacing)
+
+                // Product name
+                VStack(alignment: .leading, spacing: Layout.contentVerticalSpacing) {
+                    Text(Localization.productName)
+                        .foregroundColor(.primary)
+                        .subheadlineStyle()
+                    BasicDetailRow(content: viewModel.generatedProduct?.name,
+                                   isLoading: viewModel.isGeneratingDetails)
+                }
+
+                // Product description
+                VStack(alignment: .leading, spacing: Layout.contentVerticalSpacing) {
+                    Text(Localization.productDescription)
+                        .foregroundColor(.primary)
+                        .subheadlineStyle()
+                    BasicDetailRow(content: viewModel.generatedProduct?.fullDescription,
+                                   isLoading: viewModel.isGeneratingDetails)
+                }
+
+                // TODO: update values based on product
+                // Other details
+                VStack(alignment: .leading, spacing: Layout.contentVerticalSpacing) {
+                    Text(Localization.details)
+                        .foregroundColor(.primary)
+                        .subheadlineStyle()
+                    TitleAndValueDetailRow(title: Localization.productType,
+                                           value: "Physical",
+                                           image: UIImage.productImage,
+                                           isLoading: viewModel.isGeneratingDetails,
+                                           cornerRadius: Layout.cornerRadius)
+                    .padding(.bottom, Layout.contentPadding)
+
+                    VStack(spacing: 0) {
+                        TitleAndValueDetailRow(title: Localization.price,
+                                               value: "Regular price: $15",
+                                               image: UIImage.priceImage,
+                                               isLoading: viewModel.isGeneratingDetails)
+                        Divider()
+                            .background(Color(.separator))
+                        TitleAndValueDetailRow(title: Localization.inventory,
+                                               value: Localization.inStock,
+                                               image: UIImage.inventoryImage,
+                                               isLoading: viewModel.isGeneratingDetails)
+                        Divider()
+                            .background(Color(.separator))
+                        TitleAndValueDetailRow(title: Localization.categories,
```

Comment:
Super nit: Consider adding comments/empty lines between each field for readability.

Reply:
Updated in 399404cc3eb5aeb0008811fb34eccf2a9f507501.